xml/System.IO.Enumeration/FileSystemName.xml

```diff
@@ -87,7 +87,17 @@
         <summary> Verifies if the given Win32 expression matches the given name. Supports the following wildcards: '*', '?', '&lt;', '&gt;', '"'. The backslash character '\' escapes.</summary>
         <returns>
           <see langword="true" /> if the given expression matches the given name; otherwise, <see langword="false" />.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This is based off of <xref:System.IO.PatternMatcher> used in `FileSystemWatcher`, which is based off of `RtlIsNameInExpression`, which defines the rules for matching DOS wildcards (`'*'`, `'?'`, `'<'`, `'>'`, '"').
```

Comment:
```suggestion
The syntax of the `expression` parameter is based on the syntax used by <xref:System.IO.FileSystemWatcher>, which is based on [RtlIsNameInExpression](/windows/win32/devnotes/rtlisnameinexpression), which defines the rules for matching DOS wildcards (`'*'`, `'?'`, `'<'`, `'>'`, '"').
```
PatternMatcher is an internal type, so the link is invalid.

Reply:
Thanks for catching that. I will add a couple of missing backticks in the last wildcard:

```suggestion
The syntax of the `expression` parameter is based on the syntax used by <xref:System.IO.FileSystemWatcher>, which is based on [RtlIsNameInExpression](/windows/win32/devnotes/rtlisnameinexpression), which defines the rules for matching DOS wildcards (`'*'`, `'?'`, `'<'`, `'>'`, `'"'`).
```